### PR TITLE
Update CASP installation workflow

### PR DESCRIPTION
**Pull request by kravciak**
VMX: http://dhcp91.suse.cz/tests/3445
ISO: http://dhcp91.suse.cz/tests/3447
ISO: http://dhcp91.suse.cz/tests/3444 - with features
Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/280

### CODE REVIEW

File: products/casp/main.pm [hunk 1]
```diff
@@ -32,26 +32,42 @@ my $distri = testapi::get_var("CASEDIR") . '/lib/susedistribution.pm';
 require $distri;
```
Comment (richiejp) [inline products/casp/main.pm]: Just a thought; seems a bit strange to be doing this, maybe a bit dangerous as well

File: products/casp/main.pm [hunk 2]
```diff
@@ -32,26 +32,42 @@ my $distri = testapi::get_var("CASEDIR") . '/lib/susedistribution.pm';
...
+# Simplified workflow - fate#321754
+# https://trello.com/c/JKCIbUbv/778-5-casp-simplified-workflow
 sub load_inst_tests() {
-    loadtest "installation/welcome";
+    # Registration
     if (get_var('SCC_REGISTER', '') eq 'installation') {
         loadtest "installation/scc_registration";
     }
     else {
         loadtest "installation/skip_registration";
```
Comment (richiejp) [inline products/casp/main.pm]: minor nit; it would be more correct to use single quotes

File: products/sle/main.pm [hunk 3]
```diff
@@ -386,9 +386,7 @@ sub install_this_version {
 }
 
 sub load_inst_tests() {
-    if (!is_casp) {
```
Comment (richiejp) [inline products/sle/main.pm]: Can you remove this sub from sle/main.pm? `is_casp` that is.

File: products/casp/main.pm (hunk 1 above)
Comment (okurz) [inline products/casp/main.pm]: but necessary to initialize the distribution object for os-autoinst. Yeah, it looks a bit weird. If you have an idea how to improve, go ahead :-)

File: tests/installation/bootloader_svirt.pm [hunk 4]
```diff
@@ -258,19 +266,41 @@ sub run() {
...
+        #            type_string "echo -en '\\033[K' > \$pty\n";                     # end of line
+        #            type_string "echo -en ' $cmdline' > \$pty\n";
+        #            type_string "echo -en ' xen-fbfront.video=32,1024,768' > \$pty\n";    # set kernel framebuffer
+        #        }
+        #        else {
+        #            for (1 .. 2) { type_string "echo -en '\\033[B' > \$pty\n"; }    # four-times key down
+        #            type_string "echo -en '\\033[K' > \$pty\n";                     # end of line
+        #            type_string "echo -en ' $cmdline' > \$pty\n";
+        #            type_string "echo -en ' xen-fbfront.video=32,1024,768' > \$pty\n";    # set kernel framebuffer
+        #        }
```
Comment (okurz) [inline tests/installation/bootloader_svirt.pm]: Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

**Comment by mvidner** [comment]
Apparently `tidy` [cannot find its config file](https://travis-ci.org/os-autoinst/os-autoinst-distri-opensuse/builds/184600755#L346):
```
cannot find file given with -pro=.../.perltidyrc: No such file or directory
```

File: products/casp/main.pm (hunk 1 above)
Comment (kobliha) [inline products/casp/main.pm]: Guys, if you want to change how this is done, then I believe it should not block this PR. The PR itself does not change the initialization itself.

Comment (okurz) [inline products/casp/main.pm]: no, it's not blocking, was just a side-note.

**Comment by okurz** [comment]
Please [squash commits](https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history) and rebase on the recent style check updates. That should fix the problem as observed in travis.

File: products/sle/main.pm (hunk 3 above)
Comment (kravciak) [inline products/sle/main.pm]: done

**Comment by kravciak** [comment]
Review comments should be fixed, please check

**Comment by okurz** [comment]
Please state commit message subject line in imperative mood. Also see http://chris.beams.io/posts/git-commit/

**Comment by kravciak** [comment]
Fixed, thanks for the link, I was wondering how to write them

**Pull request merged**